src/applications/vaos/reducers/appointments.js

```diff
@@ -183,6 +193,100 @@ export default function appointmentsReducer(state = initialState, action) {
         future: null,
         futureStatus: FETCH_STATUS.notStarted,
       };
+    case FETCH_EXPRESS_CARE_WINDOWS:
+      return {
+        ...state,
+        expressCare: {
+          ...initialState.expressCare,
+          windowsStatus: FETCH_STATUS.loading,
+        },
+      };
+    case FETCH_EXPRESS_CARE_WINDOWS_SUCCEEDED: {
+      const { facilityData, nowUtc } = action;
+      const times = facilityData
+        .reduce(function(arr, row) {
```

Comment:
Reduce is fine, but you can also flatten an array of arrays like this with `[].concat(...facilityData)`